### ISSUE TITLE
Hello! Found a problem? Code block theme style and code block background color cannot be customized 

### ISSUE THREAD

**Issue by zhouxinghong** [RickStrahl/MarkdownMonster#995]
Hello! Found a problem? Code block theme style and code block background color cannot be customized
System: windows 10
Version: 2.7.10.2
Please fix this issue in time. Thanks a lot!

**Comment by zhouxinghong** [comment]
For theme customization I have a question?
How to define foreground color, background color, and block font color in the editor?Does Markdown Monster have a right click Inspect function for browsers? 
![2023-01-05_161435](https://user-images.githubusercontent.com/54253524/210732970-3311e41d-10ad-426b-8b09-cc25916b5cab.png)

**Comment by RickStrahl** [comment]
Yes - enable preview developer tools or Preview developer tools on startup.

![image](https://user-images.githubusercontent.com/1374013/211127039-4414c576-1144-413f-a424-42716d7a51da.png)

**Comment by zhouxinghong** [comment]
Thank you very much!
